administrator/language/en-GB/lib_joomla.ini

```
@@ -379,8 +379,11 @@ JLIB_HTML_BATCH_MOVE="Move"
 JLIB_HTML_BATCH_MOVE_QUESTION="Action to Perform"
 JLIB_HTML_BATCH_NO_CATEGORY="- Don't copy or move -"
 JLIB_HTML_BATCH_NOCHANGE="- Keep original Access Levels -"
-JLIB_HTML_BATCH_TAG_LABEL="Add Tag"
+JLIB_HTML_BATCH_TAG_ADD="Add"
+JLIB_HTML_BATCH_TAG_ADDREMOVE_QUESTION="Action to Perform"
+JLIB_HTML_BATCH_TAG_LABEL="Add or remove a Tag"
```

```suggestion
JLIB_HTML_BATCH_TAG_LABEL="Add or Remove Tag"
```

See `JLIB_HTML_BATCH_MENU_LABEL="Copy or Move to Category"` for consistency with capitalization.